etc/profile-a-l/cargo.profile

```diff
@@ -0,0 +1,80 @@
+# Firejail profile for cargo
+# Description: The Rust package manager
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include cargo.local
+# Persistent global definitions
+include globals.local
+
+ignore noexec ${HOME}
+ignore noexec /tmp
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}
+
+noblacklist ${HOME}/.cargo/credentials
+noblacklist ${HOME}/.cargo/credentials.toml
+
+# Allows files commonly used by IDEs
+include allow-common-devel.inc
```

Comment:
There's this note in the [cargo book](https://doc.rust-lang.org/cargo/reference/config.html):
```
Note: Cargo also reads config files without the .toml extension, such as .cargo/config. Support for the .toml
extension was added in version 1.39 and is the preferred form. If both files exist, Cargo will use the file
without the extension.
```
Currently we have mixed support for ${HOME}/.cargo/{config,crates.toml} in allow-common-devel.inc and disable-programs.inc. Perhaps we could cover both formats? I mean, add ${HOME}/.cargo/{config.toml,crates} to both files...